### DISCUSSION TITLE
python海象運算子

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#401] [General]
上課老師有提到幾種基本的運算子，但是對於海象運算子的介紹比較少一點，因此我上網查詢了相關的資料，希望可以幫到大家
[https://www.gushiciku.cn/pl/pf7m/zh-tw](https://www.gushiciku.cn/pl/pf7m/zh-tw)

**Comment by ThomasMeng0515** [comment]
老師上課的時候說先暫時略過海象運算子，之後才會再詳細介紹。

**Comment by chia-kai-chang** [comment]
@hapiwangy 能在文章中簡述連結內容的觀後心得、使用心得嗎? 嘗試提出你的觀點。😂

**Reply by hapiwangy** [reply]
在看完文章和實際操作過後，我發現了以下幾點。首先，就如同文章所提到的，海象運算子讓我們在程式碼的閱讀上清楚許多，除了表示方式更好理解之外，也把程式碼的長度縮減，減少我們出錯的機會。另外，海象運算子可以做到類似C++語言的do-while迴圈的功能，而python本身是沒有相關語法的，這對於還在由C++轉換成python的過程中的我來說感到十分親切和實用

**Comment by pekoranoinu** [comment]
這些東西對我來說有點深奧，但是感謝你的分享。

**Comment by lx3211** [comment]
看起來是可以實現同時建立新變數和進行簡單比較？

**Comment by RogelioKG** [comment]
只有我覺得每次看到海象運算子都覺得海象運算子很可愛嘛？
:=

**Comment by brian431** [comment]
這是個偶爾能派上用場的運算子，而且取名十分貼切。個人認為他與海象的相似度有個76%。
